diff --git a/android/src/main/java/com/reactnativemenu/MenuView.kt b/android/src/main/java/com/reactnativemenu/MenuView.kt
index 7841d0c7f0a7ba8576013ea632ed042ee3b056e2..17ed7c6491b60e87fe829fb3f8841424617b4cfb 100644
--- a/android/src/main/java/com/reactnativemenu/MenuView.kt
+++ b/android/src/main/java/com/reactnativemenu/MenuView.kt
@@ -53,7 +53,7 @@ class MenuView(private val mContext: ReactContext) : ReactViewGroup(mContext) {
     updateTouchDelegate()
   }

-  override fun onInterceptTouchEvent(ev: MotionEvent?): Boolean {
+  override fun onInterceptTouchEvent(ev: MotionEvent): Boolean {
     return true
   }

